packages/editor/src/browser/editor-widget.ts

```diff
@@ -84,4 +84,8 @@ export class EditorWidget extends BaseWidget implements SaveableSource, Navigata
         this.editor.restoreViewState(oldState);
     }
 
+    get onDispose(): Event<void> {
```

Comment:
I would be fine to move it to `BaseWidget` instead of using `disposed.connect` thing.

Reply:
I don't understand how moving anything to BaseWidget would help here.

Reply:
ok, i meant  it is more useful api than `disposed.connect`, would be nice to have in the base, but for sure out of scope of this PR